myip.go

```diff
@@ -49,3 +50,24 @@ func GetIP() (net.IP, error) {
 
 	return nil, fmt.Errorf("Unable to find IP")
 }
+
+// GetIPUsingIfConfig_Me uses the ifconfig.me web service to get your public IP. It returns your public IP or an error.
+func GetIPUsingIfConfig_Me() (net.IP, error) {
+	resp, err := http.Get(ifconfig.me)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
```

Comment:
Consider checking the HTTP response code before trying to read the body of the request. For example it might return a 404 and if you don't check here you might end up with a body that says something like: "The page couldn't be found". Obviously no IP address would be returned in that case.

Reply:
Added the check for Status code.